MediaPlayer/MediaSlider.module.less

```diff
@@ -0,0 +1,41 @@
+@import "../styles/mixins.less";
+@import "../styles/variables.less";
+@import "../styles/skin.less";
+
+.mediaSlider {
+	cursor: pointer;
```

Comment:
The media slider seems too thick compared to the guide. WDYT?

Reply:
Added a height of 6px to correspond to GUI guide.